pyproject.toml

```diff
@@ -152,6 +152,8 @@ select = [
     "RET",
     "RSE",
     "SIM",
+    "TCH",
+    "TID",
```

Comment:
TID: disallow `import ..something` relative imports (parent).
TCH: flag when an import is only used for type checking and enforce it being put under `if TYPE_CHECKING:`